### PR TITLE
feat: repayment email settings update

### PR DESCRIPTION
**Pull request by roger-in-kiva**
- repayment email settings update

Screenshot 2024-06-04 at 12 31 05
Screenshot 2024-06-04 at 12 31 16

### CODE REVIEW

File: src/pages/Settings/EmailSettings.vue [hunk 1]
```diff
@@ -627,8 +679,6 @@ export default {
 								}
 							}
 						`,
-						awaitRefetchQueries: true,
```
Comment (dyersituations) [inline src/pages/Settings/EmailSettings.vue:630]: Why were these two lines removed?

Comment (roger-in-kiva) [reply src/pages/Settings/EmailSettings.vue:630]: Because we have the mutation below doing the same.

Comment (dyersituations) [reply src/pages/Settings/EmailSettings.vue:630]: Right, but these are different mutations. Don't we also need to refetch after unsubscribing here?

Comment (roger-in-kiva) [reply src/pages/Settings/EmailSettings.vue:630]: Refetching is to get the updated data on the page, `pageQuery` and show it as mutations are done.
Unsubscribing implies repayments too, so for the case `Only legally required emails` we need the second mutation if we have a different value than default for repayments, so we need to refetch the page data once. makes sense?

Comment (dyersituations) [reply src/pages/Settings/EmailSettings.vue:630]: Oh I clicked to expand the methods and see that both mutations are in the same method, makes sense, I missed that before.

**Review by dyersituations — APPROVED** [review]

**Review by mcstover — APPROVED** [review]

**Pull request merged**